### ISSUE TITLE
what are the final parts that go into the rendering step?

### ISSUE THREAD

**Issue by jread-usgs** [USGS-R/meddle#5]
E.g., do we have plain text files that are used together to create the compliant xml?
 - mustache template for the xml, and a yaml to fill in all of the info?

Or, is it kept in R? 
 - xml2 package builds the metadata according to a list

I would suggest the former, because those two files could be used to render in any other language, and could be ( 😟  ) hand edited if necessary 

Thoughts @berdaniera?

**Comment by berdaniera** [comment]
Yep, I like the first approach. And the processing of those can be called from a function, maybe if all of the info is stored in the same folder, just call the directory and render it.

**Comment by jread-usgs** [comment]
Sounds like we converged on a hybrid:

mustache template for the xml, and then the global.yaml describes all of the manual fields and points to attribute tables for attributes. Optional `...` to add more arguments to the rendering step. Does that match you memory @jiwalker-usgs and @berdaniera ?

**Comment by jiwalker-usgs** [comment]
Sounds like the makings of a marquee package to me :+1:
